test/Room.test.ts

```diff
@@ -0,0 +1,93 @@
+import RoomHelper from '../src/helpers/RoomHelper';
+import { RoomConfiguration } from '../src/models/RoomConfiguration';
+import { Room } from '../src/models/Room';
+import SchemeHelper from '../src/helpers/SchemeHelper';
+import { InMemoryDatabase } from '../src/database/inMemoryDatabase';
+
+
+function createMock(returnValue: any) {
+    return {
+        createScheme: () => {
+         return returnValue
+        }
+    } 
+}
+jest.mock("uuid/v1", () => {
+    return jest.fn(() => {
+         let value = 1;
+        return value++;
```

Comment:
Een v1 UUID is een `string`, je kan hier eenvoudig een `string` returnen door `String(value++)` te returnen. Dit houdt de mock enigszins consistent met de realiteit.